docs/source/install.rst

```diff
@@ -1,15 +1,51 @@
 Install Dask
 ============
 
-You can install dask using ``conda``::
+Dask is available to Anaconda users via either the conda or pip commands, both of which are included in Anaconda. If you do not already have Anaconda, `download Anaconda now <https://www.continuum.io/downloads>`_.
+
+Dask can then be installed using ``conda``::
 
     conda install dask
 
-You can install dask using ``pip``::
+Or dask can be installed with ``pip``::
 
     pip install dask
 
-Dask also ships with Anaconda so you may have it already.
+
+Test dask install
+-----------------
+
+Because dask is a Python library, rather than a graphical program or command line program, to test that dask is working you can run a small Python program. 
```

Comment:
This section seems extraneous and distracting to me.  Most of the intended users of dask intend to use Python libraries.